app/src/renderer/components/common/TmSessionSignUp.vue

```diff
@@ -131,25 +131,6 @@
             type="required"
           />
         </tm-form-group>
-        <tm-form-group
-          :error="$v.fields.errorCollection.$error"
-          field-id="error-collection"
-          field-label
-        >
-          <div class="tm-field-checkbox">
-            <div class="tm-field-checkbox-input">
-              <input
-                id="error-collection"
-                v-model="fields.errorCollection"
-                type="checkbox"
-              >
-            </div>
-            <label class="tm-field-checkbox-label" for="error-collection">
-              I'd like to opt in for remote error tracking to help improve
```

Comment:
why are we removing the checkbox ?

Reply:
I intended to simplify the cookie process for the user. We are still transparent about what happens if the user accepts it.